interfaces/apparmor/template.go

```diff
@@ -386,6 +386,15 @@ var defaultTemplate = []byte(`
   /sys/class/ r,
   /sys/class/**/ r,
 
+  # Workaround https://launchpad.net/bugs/359338 until upstream handles
+  # stacked filesystems generally.
+  # encrypted ~/.Private and old-style encrypted $HOME
+  @{HOME}/.Private/ r,
+  @{HOME}/.Private/** mrixwlk,
+  # new-style encrypted $HOME
+  @{HOMEDIRS}/.ecryptfs/*/.Private/ r,
+  @{HOMEDIRS}/.ecryptfs/*/.Private/** mrixwlk,
```

Comment:
@jdstrand to answer your question: this works fine on a non-encrypted $HOME. However, things have changed a little since that bug was logged: since the `configure` hook now runs upon initial install, I get these errors when running `snap try` instead of `snap set`. `strace`ing just straces the snap client, where we'd actually want to strace snapd itself. I'm not sure how to do that.